package/gluon-iptables-clamp-mss-to-pmtu/files/lib/gluon/mesh-vpn/iptables-mss.rules

```diff
@@ -1,3 +1,3 @@
 *mangle
--A FORWARD -o mesh-vpn -p tcp -m tcp --tcp-flags SYN,RST SYN -d 64:ff9b::/96 -j TCPMSS --clamp-mss-to-pmtu
```

Comment:
is there an explanation on the removal of this destination part? maybe i'm just missing it.

Reply:
clamping should be done on all traffic passing through vpn interfaces, not just on traffic destined for nat64. When introducing the package I wanted to keep the scope of the rule as small as possible. During usage I had to learn & accept that the initial scope was too small, thus removing the destination.